aptos-move/framework/move-stdlib/sources/bit_vector.move

```diff
@@ -192,7 +192,7 @@ module std::bit_vector {
             }) {
                 let elem = vector::borrow_mut(&mut self.bit_field, i);
                 *elem = false;
-                i = i + 1;
+                i += 1;
```

Comment:
We can use `self.bit_field[i] = false;` in the code above.

Reply:
Also, there are multiple places where we can upgrade with Move 2 syntax:
1. `vector::length` towards the beginning of the function can be replaced with receiver style.
2. Multiple places in this function where `unset(..)` -> `self.unset(..)`.
3. `i = i + 1` towards the end of this function to `i += 1`.

Reply:
I didn't touch this one because it's used for verification, and specs were happening funny with receiver syntax.

Unless I'm interpreting this wrong

Reply:
looks like it's fine as long as it's not in a spec block. changing now